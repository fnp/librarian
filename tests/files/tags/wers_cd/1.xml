<naglowek_osoba>MARYNA</naglowek_osoba>
<kwestia>
<strofa>

    <wers_cd>Więc?</wers_cd>

</strofa>
</kwestia>

<naglowek_osoba>POETA</naglowek_osoba>
<kwestia>
<strofa>
           <wers_cd>Sztuka dla sztuki.</wers_cd>
</strofa>
</kwestia>

<naglowek_osoba>MARYNA</naglowek_osoba>
<kwestia>
<strofa>Zawrót głowy, wielka chwała;/
niech pan sztuki płata różne,/
bylebym ja spokój miała.
</strofa>
</kwestia>
