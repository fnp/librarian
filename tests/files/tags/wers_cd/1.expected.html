<div id="book-text">
<h4 id="i1">MARYNA</h4>
<div class="kwestia"><div class="stanza" id="i2">
<a href="#f1" class="wl-num">1</a><div class="wl verse verse-cont" id="f1">Więc?</div>

</div></div>
<h4 id="i3">POETA</h4>
<div class="kwestia"><div class="stanza" id="i4">
<div class="wl verse verse-cont" id="f2">Sztuka dla sztuki.</div>
</div></div>
<h4 id="i5">MARYNA</h4>
<div class="kwestia"><div class="stanza" id="i6">
<div class="wl verse" id="f3">Zawrót głowy, wielka chwała;</div>
<div class="wl verse" id="f4">niech pan sztuki płata różne,</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">bylebym ja spokój miała.</div>
</div></div>
</div>
